core/MODULE.bazel

```diff
@@ -5,5 +5,3 @@ module(
 
 bazel_dep(name = "platforms", version = "0.0.4")
 bazel_dep(name = "bazel_skylib", version = "1.0.3")
-bazel_dep(name = "rules_sh", version = "0.3.0")
-bazel_dep(name = "stardoc", version = "0.5.1", repo_name = "io_bazel_stardoc", dev_dependency = True)
```

Comment:
I am wondering if this is actually correct to remove these lines. Actually, this module still depends on these other modules since it still loads the dependencies from the new location.

Reply:
Yes, it seems like unless the tests are a separate module we'd always need to bring those in.